packages/composites/multi-address-list/README.md

```diff
@@ -0,0 +1,101 @@
+|    ✅     |    anchorage.vaultId    | `vaultID` |                                                                    customerId                                                                     |  string  |                                                 |           |            |                |
+|           |    anchorage.chainId    |           |                                                           The ID of the chain to return                                                           |  string  |              `mainnet`, `testnet`               | `mainnet` |            |                |
+|           |    anchorage.network    |           |                                                               The network to return                                                               |  string  |                                                 | `bitcoin` |            |                |
+|           |   anchorage.endpoint    |           |                                                        Endpoint name to make a request to                                                         |  string  |                                                 | `wallet`  |            |                |
+|    ✅     |          bitgo          |           |                                                         Input parameters for the Bitgo EA                                                         |  object  |                                                 |           |            |                |
+|    ✅     |       bitgo.coin        |           |                                                  A cryptocurrency symbol or token ticker symbol                                                   |  string  |                                                 |           |            |                |
+|           |      bitgo.chainId      |           |                                                           The ID of the chain to return                                                           |  string  |              `mainnet`, `testnet`               | `mainnet` |            |                |
+|           |      bitgo.network      |           |                                                               The network to return                                                               |  string  |                                                 | `bitcoin` |            |                |
+|    ✅     |      bitgo.reserve      |           |                     Used to select {$reserve}_API_KEY {$reserve}\_API_ENDPOINT {$reserve}\_API_LIMIT in environment variables                     |  string  |                                                 |           |            |                |
```

Comment:
I'm not a fan of this description carry over, similar for cbp

But for the sake of this PR, let's ignore it